flink/flink-1.14/src/main/java/io/tidb/bigdata/flink/connector/utils/FilterPushDownHelper.java

```diff
@@ -79,11 +79,18 @@
 
   private final TiTableInfo tiTableInfo;
   private final Map<String, org.tikv.common.types.DataType> nameTypeMap;
+  private final boolean isSupportEnumPushDown;
```

Comment:
if we pass the vesion. Is use a method to judge supportEnumPushDown better?

Reply:
If using the method, not variable, Every time when resolving expressions, we need to do judgments repeatedly, which is also a cost. 
@humengyu2012 What do you think

Reply:
Resolving expressions only occurs when jobgraph is generated. That means resolving expressions will only happen once for every flink job but there may be a lot of filters here(a lot of overhead). Here is a trade-off:
1. Keep the min version of tikvs, `tiKVVersions -> minVersion`;
2. Add a new method just compare 5.1.0 and minVersion for enum type.
It would be better if we have more types to be compatible with such as json, set in th future.